pkg/crypto/keys.go

```diff
@@ -49,7 +28,7 @@ func GetPublicKeysFromPrivateKey(privateKey *ecdsa.PrivateKey) ([]byte, string,
 		return nil, "", errors.New("error casting public key to ECDSA")
 	}
 
-	publicKeyDer := elliptic.Marshal(secp256k1.SECP256K1(), publicKeyECDSA.X, publicKeyECDSA.Y)
+	publicKeyDer := ethcrypto.FromECDSAPub(publicKeyECDSA)
```

Comment:
this method is simply
`func FromECDSAPub(pub *ecdsa.PublicKey) []byte {
	if pub == nil || pub.X == nil || pub.Y == nil {
		return nil
	}
	return elliptic.Marshal(S256(), pub.X, pub.Y)
}`

maybe we can inline it in our SDK being just a validation on the elliptic marshaller?

Reply:
But: `S256()` is something which belongs to `go-ethereum` - what would the benefit be of in-lining this?

Reply:
gotcha - i missed it